articles/install-guide/python.md

```diff
@@ -14,11 +14,11 @@ The [qsharp package for Python](https://pypi.org/project/qsharp/) provides inter
 ## Pre-requisites ##
 
 To get started with the `qsharp` package, you'll need the following prerequisites:
-- IQ#.
-- Python 3.6 or later,
-
-Make sure you have followed the installation steps in the [getting started guide](xref:microsoft.quantum.install) for IQ#.
+- **.NET Core 2.2 or later**,
```

Comment:
Users will need the .NET Core SDK in particular (there's also the runtime, but we need the compiler infrastructure that comes with the SDK); would it be good to note that here?

Reply:
Added "SDK"